sbt-play-ebean/src/main/scala/play/ebean/sbt/SbtEbean.scala

```diff
@@ -108,7 +107,7 @@ object SbtEbean extends AutoPlugin {
     // code to load the config as it would be loaded in production
     def withClassLoader[T](block: ClassLoader => T): T = {
       val classpath = unmanagedResourceDirectories.value.map(_.toURI.toURL) ++ dependencyClasspath.value.map(_.data.toURI.toURL)
-      val classLoader = new URLClassLoader(classpath.toArray, null)
+      val classLoader = new URLClassLoader(classpath.toArray, this.getClass.getClassLoader)
```

Comment:
The actual fix had nothing to do with classloader's classpath :-) The classpath was good, the problem was that an exception could be thrown during classloading. Here is a pointer to the actual fix https://github.com/playframework/play-ebean/pull/34

Quite instructive ticket. Thanks @jsuereth for your help, really appreciated it!